### ISSUE TITLE
[Bug]: iam_root_mfa_enabled check reports fail when root user MFA is set. 

### ISSUE THREAD

**Issue by rucheat** [toniblyx/prowler#1877]
Hi,

What happened wrong with MFA check?

The **iam_root_mfa_enabled** check fails, when I have a MFA set on root account it still report as FAIL. 

**How to reproduce it**
1. Set up MFA on root user.
2. run the check using following command - python prowler.py aws --checks iam_root_mfa_enabled

**Steps with Screenshots**
On IAM page:
![image](https://user-images.githubusercontent.com/1405686/217926691-7ec15d6f-0fdd-4143-b0ef-8f02160d9f6c.png)

Only **iam_root_mfa_enabled** check result-
![image](https://user-images.githubusercontent.com/1405686/217927084-0eeaf3a0-62d0-457e-8f51-6dcddd8e86a8.png)

aws cli output - 
![image](https://user-images.githubusercontent.com/1405686/217927403-6b8d2496-12ad-4959-b852-e2fd4f8d5ca7.png)

**Expected behavior**
The check should mark this as passed.

**From where are you running Prowler?**
Resource: IAM
OS: Local Kali VM
Prowler Version [./prowler --version]: 3.1.4

**Additional context**
N/A

**Comment by n4ch04** [comment]
Hi @rucheat we are going to review this and let you know, thanks for using prowler !!!

**Comment by n4ch04** [comment]
Hi @rucheat right now that check gets the information from the credential report, the result is weird, you can test it retrieving your credential report with:
` aws iam get-credential-report`
and checking if the field `mfa_active` for the account `<root_account>` is set to `true` or `false`.
It seems that your credential report is not synchronised with the latest changes you made with the mfa device, can you relaunch the check again to ensure the issue doesn't persist ?

**Comment by rucheat** [comment]
There was no credential report generated so I got error. 
![image](https://user-images.githubusercontent.com/1405686/218158247-b4fe32e3-121d-4083-840d-606f057aa2b3.png)

I got it working and got correct output (PASS) once I generated credential report using following command -
`aws iam generate-credential-report`

I am not sure, it this a pre-requisite to generate credential before auditing using prowler?

It would be great if it is possible to generate report by prowler and then run the prowler audit (may be for all other checks as well to get accurate results).

Feel free to close this ticket.
